articles/architecture-scenarios/implementations/mobile-api/mobile-implementation-android.md

```diff
@@ -0,0 +1,1260 @@
+---
+description: The Android implementation of the API for the Mobile + API architecture scenario
+url: /architecture-scenarios/application/mobile-api/mobile-implementation-android
+toc: true
+---
+
+# The Mobile Implementation in Android
+
+This document is part of the [Mobile + API Architecture Scenario](/architecture-scenarios/application/mobile-api) and it explains how to implement the Mobile application in Android. Please refer to the scenario for information on the implemented solution.
+
+::: note
+The full source code for the Node.js API implementation can be found in [this GitHub repository](https://github.com/auth0-samples/auth0-pnp-abc-timesheets/tree/master/timesheets-mobile/android).
+:::
+
+## 1. Set Up the Application
+
+### System Requirements
```

Comment:
I'm wondering if instead of that we can use something similar to the quickstarts, both for the github URL and the system requirements:

![image](https://user-images.githubusercontent.com/11715799/29119657-bc8b6596-7d0f-11e7-9c2e-5eac42805616.png)

@jerriep wdyt?

Reply:
@erjohnson Do this, slack me if you don't know how